ui/components/app/import-token/token-search/token-search.component.js

```diff
@@ -44,7 +44,7 @@ export default function TokenSearch({
 
   const filteredTokenList = useMemo(() => {
     if (isTokenNetworkFilterEqualCurrentNetwork) {
-      const dataObject = tokenList?.[chainId]?.data;
+      const dataObject = tokenList?.[chainId]?.data || {};
```

Comment:
This was added recently and was making e2e tests sometime fail when clicking the import button before tokenList was fully loaded. With `dataObject === undefined`, the next statement fails.